internal/transport/http2_server.go

```diff
@@ -331,6 +333,18 @@ func (t *http2Server) operateHeaders(frame *http2.MetaHeadersFrame, handle func(
 		method:         state.data.method,
 		contentSubtype: state.data.contentSubtype,
 	}
+
+	if profiling.IsEnabled() {
+		s.stat = profiling.NewStat("server")
+		// See comment above StreamStatMetadataSize definition for more information
+		// on this encoding.
+		s.stat.Metadata = make([]byte, profiling.StreamStatMetadataSize)
+		binary.BigEndian.PutUint64(s.stat.Metadata[0:8], t.connectionID)
```

Comment:
Sorry: I was suggesting consts for both the 8 and 12 magic numbers (where 12 potentially = the 8 constant plus another 4 constant), and to use them here, also.